File: Solutions/C++/2021/J3/J3.cpp
#include <iostream>
using namespace std;

int main() {
  int n,a,b;
  string d;

  while (true){
    cin >> n;

    a = n/1000 - n/10000*10 + n/10000;

    if (n == 99999) break;

    if (a == 0) d = d;
    else if (a % 2 == 0) d = "right ";
    else d = "left ";

    cout << d << n % 1000 << endl;
  }

}

File: Solutions/Python/2021/J3.py
#This does not catch improper inputs so it will crash if so


d = ""
while True:
    x = int(input())

    if x == 99999: break

    a = int(x/1000) - int(x/10000)*10 + int(x/10000)

    if a == 0: pass
    elif (a % 2): d = "left"
    else: d = "right"

    print(d, x%1000)

